src/communication/network_interface_cl.c

```diff
@@ -11602,3 +11603,101 @@ tdes_reset_query_start_info (PT_NODE * node)
     }
 #endif
 }
+
+/* create_lob_dir - create lob dir
+ *
+ * return:
+ *
+ *   hfid(in): separate lob dir by table
+ *   attrid_arr(in): separate lob dir by column
+ *   attrid_arr_length(in): number of LOB-type columns
+ *
+ * NOTE:
+ * create_lob_dir - request to server to create lob dir
+ */
+int
+create_lob_dir (HFID * hfid, int *attrid_arr, int attrid_arr_length)
+{
+#if defined(CS_MODE)
+  int error = ER_NET_CLIENT_DATA_RECEIVE;
+  int req_error, request_size;
+  char *ptr;
+  char *request, *reply;
+  OR_ALIGNED_BUF (OR_INT_SIZE) a_reply;
+
+  reply = OR_ALIGNED_BUF_START (a_reply);
+
+  request_size = OR_HFID_SIZE + OR_INT_SIZE + (OR_INT_SIZE * attrid_arr_length);
+  request = (char *) malloc (request_size);
+
+  ptr = or_pack_hfid (request, hfid);
+  ptr = or_pack_int_array (ptr, attrid_arr_length, attrid_arr);
+
+  req_error =
+    net_client_request (NET_SERVER_CREATE_LOB_DIR, request, request_size, reply,
+			OR_ALIGNED_BUF_SIZE (a_reply), NULL, 0, NULL, 0);
+  if (!req_error)
```

Comment:
NIT:
다음과 같이 수정하는게 가독성이 더 좋을 것 같습니다.
```suggestion
  if (req_error == NO_ERROR)
```

Reply:
반영했습니다.